regtest.py

```diff
@@ -1090,6 +1091,9 @@ def test_suite(argv):
                                 with open(outfile) as f:
                                     print(f.read())
 
+                        analysis_time = analysis_start_time - time.time()
+                        suite.log.log(f"Analysis time: {analyis_time:.3f} s")
+
```

Comment:
oh, I missed the pylint error -- can you do another PR fixing the spelling of `analyis_time` in the log statement?